<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>机房失物招领系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newRecord">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>新建记录</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmRecord">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>确认记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageAdmin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>510</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>管理管理员</string>
    </property>
   </widget>
   <widget class="QPushButton" name="found">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>510</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>认领</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="recordTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1051</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
